src/PopEngine/index.ts

```diff
@@ -264,6 +229,50 @@ export class PopEngine {
     }
   }
 
+  private _maybeCreateOrReusePopover(isAlreadyShowing: boolean, targetElement: Element, pop: Pop, groupId: string): Element {
+    let container = <Element>document.querySelector('div[pop-id="' + groupId + '"]');
+    let oldPop = this.getPopFromGroupId(groupId);
+    if (oldPop && oldPop.targetEl && !pop.opts.reusePopover) {
```

Comment:
```
 +    if (oldPop && oldPop.targetEl && !pop.opts.reusePopover) {
 +      this.synchronousHidePop(oldPop.targetEl, false);
 +    }
```

Hm.. why does `pop.opts.reusePopover` play a role in determining if this pop should close or not? Also I'm wondering if this logic makes sense in this `_maybeCreateOrReusePopover` function since it neither creates or reuses a popover.